feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -269,33 +280,44 @@ func configureATE(t *testing.T, ate *ondatra.ATEDevice) []*ondatra.Flow {
 	i3.IPv4().WithAddress(ateDst2.IPv4CIDR()).WithDefaultGateway(dutDst2.IPv4)
 	i3.IPv6().WithAddress(ateDst2.IPv6CIDR()).WithDefaultGateway(dutDst2.IPv6)
 
+	t.Logf("Pushing config to ATE and starting protocols...")
+	topo.Push(t)
+	topo.StartProtocols(t)
+}
+
+var (
+	TrafficFlows = make(map[string]*ondatra.Flow)
```

Comment:
this should be of a new struct type with each flow as a field, not a map. something like

type trafficFlows struct {
  ipInIPFlow1, ipInIPFlow2, ipInIPFlow3 *ondatra.Flow
}

and then createTrafficFlows should have return type *trafficFlows

Reply:
done in new commit.